Source/SwiftLintFramework/Rules/Lint/NSObjectPreferIsEqualRule.swift

```diff
@@ -89,6 +83,14 @@ private extension FunctionDeclSyntax {
     }
 }
 
+private extension Syntax {
+    var isInObjcClass: Bool {
```

Comment:
maybe the existing rule had this problem already, but we should also check for `@objcmembers`

Reply:
You are right. It didn't work before either. Added a CHANGELOG entry for the fix.